desktop/conf.dist/hue.ini

```diff
@@ -186,6 +186,9 @@ http_500_debug_mode=false
 # Enable X-Forwarded-Host header if the load balancer requires it.
 ## use_x_forwarded_host=true
 
+# Enable X-Forwarded-For header if hive/impala requires it.
+## use_x_forwarded_for=true
```

Comment:
use better ini variable name, such as enable_xff_for_hive_impala

Reply:
+1

Reply:
renamed it to "enable_xff_for_hive_impala"